.eslintrc.json

```diff
@@ -51,7 +51,7 @@
 		"react-hooks/exhaustive-deps": "warn",
 		"semi": [2, "always"],
 		// "simple-import-sort/imports": "error",
-		"sort-keys": ["error", "asc", {"caseSensitive": true, "natural": false, "minKeys": 2}],
+		// "sort-keys": ["error", "asc", {"caseSensitive": true, "natural": false, "minKeys": 2}],
```

Comment:
Please enable this, should be turned off for individual pages instead if needed.